inc/savedsearch.class.php

```diff
@@ -1269,7 +1269,7 @@ public function setEntityRecur(array $ids, $eid, $recur) {
     *
     * @return string where clause
     */
-   public static function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val, $meta = 0) {
+   public static function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val, array &$qry_params, $meta = 0) {
```

Comment:
Signature update should be documented in changelog and PHPDoc should be updated.